checklists/avd_checklist.en.json

```diff
@@ -518,64 +302,496 @@
+            "guid": "3611c818-b0a0-4bc5-80e4-3a18a9cd289c",
+            "severity": "Medium",
+            "link": "https://docs.microsoft.com/azure/virtual-machines/disks-types"
+        },
+        {
+            "category": "Storage",
+            "subcategory": "Capacity Planning",
+            "text": "Determine which storage backend solution will be used for FSLogix Profiles",
+            "description": "Possible options are: Azure NetApp Files, Azure Files, VM based File Server. File-server it is not recommended. Azure Files Premium typically a good starting point. NetApp usually required for large scale / high-performant environment. For a detailed comparison see the article in the 'More Info' column.",
+            "guid": "ed6b17db-8255-4462-b2ae-e4553afc8339",
+            "severity": "High",
+            "link": "https://docs.microsoft.com/azure/virtual-desktop/store-fslogix-profile"
+        },
+        {
+            "category": "Storage",
+            "subcategory": "Capacity Planning",
+            "text": "Do not share storage and profiles between different Host Pools",
+            "description": "Every Host Pool should use a separate set of storage accounts/volumes (at least one) and shares. Users should have a different profile for each Host Pool since settings and configurations are specific to each Host Pool. Additionally, accessing different Host Pools at the same time can cause errors on the shared user profile VHD/X. Usage of different storage accounts/volumes for multiple shares is also recommended to scale independently.",
+            "guid": "2fad62bd-5004-453c-ace4-64d862e7f5a4",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/virtual-desktop/store-fslogix-profile"
+        },
+        {
+            "category": "Storage",
+            "subcategory": "Capacity Planning",
+            "text": "Verify storage scalability limits and Host Pool requirements",
+            "description": "As a starting point for estimating profile container storage performance requirements we recommend to assume 10 IOPS per user in the steady state and 50 IOPS per user during sign-in/sign-out. Space requirements is simply obtained based on the maximum profiles size in FSLogix per the total number of users for each Host Pool. Multiple storage accounts can be used for the same Host Pool if required.",
+            "guid": "680e7828-9c93-4665-9d02-bff4564b0d93",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/virtual-desktop/faq#what-s-the-largest-profile-size-fslogix-can-handle-"
+        },
+        {
+            "category": "Storage",
+            "subcategory": "Capacity Planning",
+            "text": "For optimal performance, the storage solution and the FSLogix profile container should be in the same data-center location.",
```

Comment:
Using "data-center location" here feels misleading. We're not really talking about data-centers, I don't think. Would it be better to say "Azure region" instead?